packages/project-config/src/builds.ts

```diff
@@ -125,16 +129,18 @@ export interface ProjectBuildOptions {
   /** Options for processing JavaScript. */
   js?: {
     /** Minify inlined and external JavaScript. */
-    minify?: boolean | {
+    minify?: boolean|{
       /** JavaScript files listed here will not be minified. */
       exclude?: string[],
     },
 
     /** Use babel to compile all ES6 JS down to ES5 for older browsers. */
-    compile?: boolean|{
-      /** JavaScript files listed here will not be compiled. */
-      exclude?: string[],
-    },
+    compile?:
+               boolean|JsCompileTarget|{
```

Comment:
Is this backwards compatible? Will there be problems when a user runs an old version of polymer cli with a string for `compile`?